dts/bindings/i2s/st,stm32-i2s.yaml

```diff
@@ -5,7 +5,7 @@ description: STM32 I2S controller
 
 compatible: "st,stm32-i2s"
 
-include: [i2s-controller.yaml, dma.yaml]
+include: [i2s-controller.yaml]
```

Comment:
we did include the dma bindings for the spi client, so why not for i2s

Reply:
dma client properties have been moved to base.yaml.
So they'll be available for all clients, no more need to include dma.yaml (which has been deleted)